src/org/folio/client/reportportal/ReportPortalClient.groovy

```diff
@@ -0,0 +1,132 @@
+package org.folio.client.reportportal
+
+import com.cloudbees.groovy.cps.NonCPS
+import groovy.json.JsonOutput
+import groovy.text.StreamingTemplateEngine
+import org.folio.shared.TestType
+import org.folio.utilities.RestClient
+
+import java.time.Instant
+
+class ReportPortalClient {
+
+  private enum ReportPortalTestType{
+    KARATE(ReportPortalConstants.KARATE_PROJECT_NAME, ReportPortalConstants.KARATE_EXEC_PARAM_TEMPLATE, TestType.KARATE),
+    CYPRESS(ReportPortalConstants.CYPRESS_PROJECT_NAME, ReportPortalConstants.CYPRESS_EXEC_PARAM_TEMPLATE, TestType.CYPRESS)
```

Comment:
We will have more project then only karate and cypress. Especially CI quality gate cypress and karate, and maybe performance and security is it will be easy to extend this later?

Reply:
I think so. Just add a new enum value and the functionality accordingly, if required.